Dockerfile

```diff
@@ -17,33 +17,50 @@ RUN . ~/venv/bin/activate;pip install -r /app/requirements.txt
 RUN . ~/venv/bin/activate;pip install coverage==3.6
 RUN . ~/venv/bin/activate;pip install gunicorn==19.7.1
 
+# Create /etc/ssl
+RUN mkdir -p /etc/ssl/
+COPY deploy/ssl/* /etc/ssl/
+VOLUME /etc/ssl
+
 # Create /deploy
 RUN mkdir -p /deploy
 COPY deploy/gunicorn.conf.py  /deploy
 COPY deploy/run.sh  /deploy
 
-# Create /etc/ssl
-RUN mkdir -p /etc/ssl/
-COPY deploy/ssl/* /etc/ssl/
-VOLUME /etc/ssl
+# Config supervisor
+COPY deploy/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 
-# Create /var/logs
-RUN mkdir -p /var/logs
-VOLUME /var/logs
+# Create log directories
+RUN mkdir -p /var/log/supervisor/
+VOLUME /var/log
 
-# Copy server code
-COPY . /app/
-WORKDIR /app/
+# Create /sshkeys
+RUN mkdir -p /sshkeys
+VOLUME /sshkeys
 
-# Initialize database & Collect static files
-RUN . ~/venv/bin/activate;make server-init server-static
+# Create /var/data
+RUN mkdir -p /var/data
+VOLUME /var/data
 
-WORKDIR /app/quipucords
+# Copy server code
+COPY . /app/
+WORKDIR /app
```

Comment:
This use to be: `WORKDIR /app/` (Just wanted to make sure the removal of / was intentional)

Reply:
The trailing slash has no effect in this case.